Atelier/Atelier/Services/InitialOperations.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Atelier.Services
+{
+    internal class InitialOperations
```

Comment:
Class is doing too much, the code might need to be separated into other classes/methods